angular.json

```diff
@@ -60,8 +60,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumWarning": "6mb",
```

Comment:
Why are these limits raised so much? What has changed that the app is now so big (and therefore heavy to load)? Have we looked into ways to make the app smaller?

Reply:
We had a look at this with @paddybasi on master before and had decided to defer this. Basically, this is the same as master. Maybe we should create a separate ticket to deal with this.